metadata.json

```diff
@@ -14,13 +14,21 @@
       "name": "puppet/archive",
       "version_requirement": ">= 2.0.0 < 7.0.0"
     },
+    {
+      "name": "puppetlabs/concat",
+      "version_requirement": ">= 1.2.5 < 8.0.0"
+    },
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.13.1 < 9.0.0"
     },
     {
       "name": "puppet/systemd",
       "version_requirement": ">= 2.10.0 < 5.0.0"
+    },
+    {
+      "name": "puppet/yum",
+      "version_requirement": ">= 5.0.0 < 6.2.0"
```

Comment:
please pin to the next major version, 7.0.0

Reply:
Done.